.storybook/config.js

```diff
@@ -0,0 +1,32 @@
+import { configure, addParameters } from '@storybook/react';
+import { DocsPage, DocsContainer } from '@storybook/addon-docs/blocks';
+import { action } from '@storybook/addon-actions';
+
+addParameters({
+    docs: {
+        container: DocsContainer,
+        page: DocsPage,
+        prepareForInline: storyFn => storyFn(),
+    },
+    options: {
+        storySort: (a, b) =>
```

Comment:
This is neat, it allows us to sort stories alphabetically

Reply:
oh nice!